internal/libwebp/include/webp.h

```diff
@@ -32,6 +32,19 @@ uint8_t* webpDecodeRGBA(
 	int* width, int* height
 );
 
+int webpDecodeGrayToSize(const uint8_t* data, size_t data_size,
+	int width, int height,
+	uint8_t* gray, size_t gray_size
```

Comment:
I think the `gray_size` is not needed.
I suggest a `widthStep` param, then we can use `webpDecodeGrayToSize` to decode a subimage to a large image buffer.